{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,24 +1,17 @@
 [tox]
-envlist = py34-coverage
+envlist = py34
 skipsdist = True
 
-[base]
-
 [testenv]
-setenv =
```

Comment:
Well the envdir is a way to set environment variables among others, you could also export them directly in your webserver and then pass them to the wsgi processes or set them in heroku, etc. I'd say the envdir is mostly useful for configuration of environment-dependent secrets, and in the case of the testing environment that's not secret, so I think it would make more sense to group these "dummy" settings either in a `settings/test.py` file, or set them in the `tox.ini` file.